account_permanent_lock_move_update/wizards/account_update_lock_date.py

```diff
@@ -6,32 +6,28 @@
 
 
 class AccountUpdateLockDate(models.TransientModel):
-
     _inherit = 'account.update.lock_date'
 
-    permanent_lock_date = fields.Date(
-        string="Permanent Lock Date", readonly=True)
+    tmp_permanent_lock_date = fields.Date(string="Permanent Lock Date")
```

Comment:
I think we should at least update the module version as you modify the field name.